STUDIO/org.openl.rules.repository.azure/src/org/openl/rules/repository/azure/AzureYamlMapperFactory.java

```diff
@@ -0,0 +1,33 @@
+package org.openl.rules.repository.azure;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.MapperFeature;
+import com.fasterxml.jackson.databind.SerializationFeature;
+import com.fasterxml.jackson.dataformat.yaml.YAMLFactory;
+import com.fasterxml.jackson.dataformat.yaml.YAMLGenerator;
+import com.fasterxml.jackson.dataformat.yaml.YAMLMapper;
+
+/**
+ * Simple YAML ObjectMapper factory for Azure
+ *
+ * @author Vladyslav Pikus
+ */
+final class AzureYamlMapperFactory {
+
+    private AzureYamlMapperFactory() {
+    }
+
+    public static YAMLMapper createObjectMapper() {
+        var factory = new YAMLFactory().disable(YAMLGenerator.Feature.WRITE_DOC_START_MARKER)
+            .enable(YAMLGenerator.Feature.MINIMIZE_QUOTES);
+        return YAMLMapper.builder(factory)
+                .serializationInclusion(JsonInclude.Include.NON_NULL)
+                .deactivateDefaultTyping()
+                .enable(SerializationFeature.INDENT_OUTPUT)
+                .enable(MapperFeature.PROPAGATE_TRANSIENT_MARKER)
+                .disable(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES)
+                .build();
+    }
```

Comment:
It seems that it is common configuration for all OpenL yaml files. Can we extract it to some util place?

Reply:
Create a module with only one bean?